tests/packages/packages/c/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
These seem a little too verbose to me for a test asset. Perhaps their contents can be limited to what's actually required/necessary?